travel.js

```diff
@@ -0,0 +1,28 @@
+const travel = [
+    const title = {
+        h3: "Travel",
+    }
```

Comment:
add a comma after your objects in your array also, push your array to the main object "intelReport" :)